### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by pmayur**
.gitignore to ignore nodemodules on npm install not present

### CODE REVIEW

**Comment by pmayur** [comment]
.gitignore to ignore nodemodules on npm install not present

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires at least one additional approval before this can be merged. 🙂

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**
